frontend/src/components/HomeFooter/index.tsx

```diff
@@ -1,10 +1,19 @@
-import { Email, GitHub, Instagram, LinkedIn } from "@material-ui/icons";
+import {
+  Email,
+  GitHub,
+  Instagram,
+  LinkedIn,
+  Timeline,
+} from "@material-ui/icons";
+// import TimelineIcon from '@material-ui/icons/Timeline';
```

Comment:
Please remove commented code if not necessary.

Reply:
Removed the same ✌